src/python/platforms/fuchsia/device.py

```diff
@@ -91,8 +91,7 @@ def qemu_setup():
 
   # Finally, launch QEMU.
   qemu_process = new_process.ProcessRunner(qemu_path, qemu_args)
-  # TODO(flowerhack): Implement teardown logic.
-  qemu_process.run()
```

Comment:
why was this removed?

Reply:
Because I implemented it :)  Added a clarifying comment to qemu_process--tl;dr, it's the responsibility of the caller to kill the process, and added the appropriate kills for each time this function's called